### PR TITLE
Initial spec

### PR DESCRIPTION
**Pull request by syg**
Trying to get this merged before July TC39 agenda deadline at July 10 10:00 PT.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  <emu-clause id="sec-array.prototype.item">
+    <h1>Array.prototype.item ( _index_ )</h1>
+    <emu-alg>
+      1. Let _O_ be ? ToObject(*this* value).
+      1. Let _len_ be ? LengthOfArrayLike(_O_).
+      1. Let _relativeIndex_ be ? ToInteger(_index_).
+      1. If _relativeIndex_ &ge; 0, then
+        1. Let _k_ be _relativeIndex_.
+      1. Else,
+        1. Let _k_ be max((_len_ + _relativeIndex_), 0).
```
Comment (ljharb) [inline spec.html:31]:
```suggestion
        1. Let _k_ be max(_len_ + _relativeIndex_, 0).
```

File: spec.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <h1>String.prototype.item ( _index_ )</h1>
+    <emu-alg>
+      1. Let _O_ be ? RequireObjectCoercible(*this* value).
+      1. Let _S_ be ? ToString(_O_).
+      1. Let _len_ be the length of _S_.
+      1. Let _relativeIndex_ be ? ToInteger(_index_).
+      1. If _relativeIndex_ &ge; 0, then
+        1. Let _k_ be _relativeIndex_.
+      1. Else,
+        1. Let _k_ be max((_len_ + _relativeIndex_), 0).
```
Comment (ljharb) [inline spec.html:50]:
```suggestion
        1. Let _k_ be max(_len_ + _relativeIndex_, 0).
```

File: spec.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      1. Return the String value containing the code unit at _k_.
```
Comment (ljharb) [inline spec.html:51]: at k in what?
this is the text `charAt` uses:
```suggestion
      1. Return the String value of length 1, containing one code unit from _S_, namely the code unit at _k_ position.
```

File: spec.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    <h1>%TypedArray%.prototype.item ( _index_ )</h1>
+    <emu-alg>
+      1. Let _O_ be the *this* value.
+      1. Perform ? ValidateTypedArray(_O_).
+      1. Let _len_ be _O_.[[ArrayLength]].
+      1. Let _relativeIndex_ be ? ToInteger(_index_).
+      1. If _relativeIndex_ &ge; 0, then
+        1. Let _k_ be _relativeIndex_.
+      1. Else,
+        1. Let _k_ be max((_len_ + _relativeIndex_), 0).
```
Comment (ljharb) [inline spec.html:69]:
```suggestion
        1. Let _k_ be max(_len_ + _relativeIndex_, 0).
```

**Comment by tabatkins** [comment]
I don't have ecmarkup set up; if you could regen the output file, Shu, I'm happy to merge. Thanks so much for getting to this. ^_^

**Comment by syg** [comment]
@tabatkins Done! Thanks for applying the review comments. BTW I haven't turned on gh-pages for the docs/ directory, don't have access to do that.

**Pull request merged**